clinica/iotools/converters/adni_to_bids/adni_utils.py

```diff
@@ -941,18 +941,11 @@ def create_adni_sessions_dict(
     # write line to get field_bids = sessions['BIDS CLINICA'] without the null values
 
     # Iterate over the metadata files
-
     for location in files:
-
         location = location.split("/")[0]
-        if path.exists(path.join(clinical_data_dir, location)):
-
-            file_to_read_path = path.join(clinical_data_dir, location)
-            cprint(f"\tReading clinical data file: {location}")
-
-            df_file = pd.read_csv(file_to_read_path, dtype=str)
+        try:
```

Comment:
I would put as few lines as possible in the try-catch and catch explicit errors. In this case, we need to catch the IOError which happens when the number of found files isn't exactly one. It could be a good idea to give a warning and continue the loop. WDYT ?

```python
try:
    df_file = load_clinical_csv(clinical_data_dir, location.split(".")[0])
except IOError as e:
    warnings.warn(e)
    continue
 df_filtered = ...
```